lib/kitchen/directions/bake_custom_sections/v1.rb

```diff
@@ -22,6 +22,18 @@ def bake(chapter:, custom_sections_properties:)
               HTML
             )
             custom_section_title_os_text.replace_children(with: title_text)
+          when 'title_document_subtitle'
+            custom_section_title = custom_section.first('h2')
+            custom_section_title_os_text = custom_section_title.first('.os-text')
+            custom_section_title_sibling = custom_section.first('h2 + div')
+            div_id = custom_section_title_sibling['id']
+            custom_section_title_sibling.trash
+            custom_section_title.append(sibling:
+              <<~HTML
+                <h3 class="os-subtitle" data-type="document-subtitle" id="#{div_id}">#{custom_section_title_os_text.text}</h3>
+              HTML
+            )
+            custom_section_title_os_text.replace_children(with: title_text)
```

Comment:
This may be extra but I was worried about catching unnessicary h3s, would a patch to recipes instead be better?

Reply:
is it necessary to add another block, or can the original `'title'` block just be modified?

Reply:
Well, we don't want to tag all the tite's subtitle sections like that, though it is otherwise identical. So far, we just want the trailblazer section's subtitle h3s to have that data type. Maybe I could pass in an extra boolean variable to switch the document-subtitles on? but it would only apply in this use case

Reply:
I could be missing a neater solution :T